src/Uno.UWP/System/Profile/AnalyticsVersionInfo.wasm.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using static Uno.Foundation.WebAssemblyRuntime;
+
+namespace Windows.System.Profile
+{
+	public partial class AnalyticsVersionInfo
+	{
+		private const string BrowserTypeFallback = "Browser";
+		private const string BrowserVersionFallback = "Unknown";
+		private const string JsType = "Windows.System.Profile.AnalyticsVersionInfo";
+
+		internal AnalyticsVersionInfo()
+		{
+		}
+
+		public string DeviceFamily => $"{GetBrowserName()}.{AnalyticsInfo.DeviceForm}";
+
+		public string DeviceFamilyVersion => GetUserAgent();
+
+		private string GetBrowserName()
+		{
+			var detectedBrowser = InvokeJS($"{JsType}.getBrowserName()");
```

Comment:
You should use `+` (string concatenation) instead of string interpolation to let the compiler do it at compile time instead of computing during runtime.

(I'm pretty sure the compiler can't optimize string interporation...)

Reply:
You are completely right. It is even safer, as string interpolation uses current culture, I keep forgetting that